kura/test/org.eclipse.kura.linux.net.test/src/test/java/org/eclipse/kura/linux/net/util/DhcpLeaseToolTest.java

```diff
@@ -0,0 +1,62 @@
+package org.eclipse.kura.linux.net.util;
```

Comment:
Please follow the [guide](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md) for the test development. Recently we switched to the Gerkin style for new tests. It follows the given-when-then pattern for testing features.

Reply:
OK.